### ISSUE TITLE
[Organizer Request] Add Yongkang He as the co-organizer for London

### ISSUE THREAD

**Issue by yongkanghe** [cncf/communitygroups#121]
### Name of chapter inquiring about

Cloud Native London

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Yongkang He
[yhe@yongkang.com.au](mailto:yhe@yongkang.com.au)
K8SUG
https://github.com/yongkanghe
https://linkedin.com/in/yongkanghe
yongkang
https://community.cncf.io/u/mvs8uf/#/about

### Tell us why you want to be added or removed from the chapter

London community is not active for a long time. I had very strong experience to help the community. I'd love to make it actively running.

### Code of Conduct

- [X] I agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizer resides

- [X] I confirm I am local to or live near the city of the above-proposed chapter (do not mark if irrelevent to request)

### Organizers companies/affiliations

- [ ] I come from xyz company which is differing from that of the current organizers (do not mark if irrelevent to request)

### Inclusive

- [X] I agree. To assure this, I have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) (do not mark if irrelevent to request)

### Vendor Neutral

- [X] I have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and I agree the main content of our meetups will remain vendor-nuetral (do not mark if irrelevent to request)

### Remain an active chapter

- [X] I understand the activity level standards of all CNCF chapters (do not mark if irrelevent to request)

### Collaboration with other chapters

- [X] I acknowledge the CNCF right to approve additional chapters in my area based on this criteria (do not mark if irrelevent to request)

### Aim to have a diverse representation of speakers in your meetups

- [X] I will do my best to host diverse speakers throughout the year (do not mark if irrelevent to request)

**Comment by yongkanghe** [comment]
@AudMonte01 @k8tgreenley

**Comment by AudMonte01** [comment]
Hi @g-nieboer - do you agree to Yonkanghe being added to this chapter, and it turning into a K8SUG chapter?

Also, with nearly 9 million people living in London, we recommend calling this something more specific, like a borough, district, neighborhood, or suburb. For example, we previously had a chapter (not inactive) that was called Birmingham, London.

**Comment by g-nieboer** [comment]
Sounds like his current community is K8SUG (Singapore) but wouldn't it make
more sense to keep it as K8LON? Happy for Yonkanghe to lead this and make
the community active again.

**Comment by AudMonte01** [comment]
The acronym actually stands for K8s User Group. We would need to get more specific with location is my only concern.

This is something we are currently adjusting for New York, for example "Midtown" since there are multiple groups emerging  Manhattan alone, which has over 1.5 million people.

**Comment by g-nieboer** [comment]
Then that sounds good to me!

**Comment by AudMonte01** [comment]
Hi folks -after speaking with Yonkeng, he confirmed the K8SUG brand is to be separate from the CNCF chapters, and CNCF agrees with this method, as the approach in regards to content is not that of our chapter's standards. Additionally, Yonkeng is not local to London, so we recommended he be more of an advisor to folks (if they wish to accept his guidance), but not to directly manage chapters in cities he does not reside in. That said, this issue/ticket is now closed.

@g-nieboer - we can discuss a name change of the chapter with Syed over email.
